src/wp-includes/script-loader.php

```diff
@@ -2966,6 +2966,75 @@ static function () use ( $style ) {
 	);
 }
 
+/**
+ * Fetches, processes and compiles stored core styles, then combines and renders them to the page.
+ * Styles are stored via the style engine API.
+ *
+ * See: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-style-engine/
```

Comment:
I think this should use `@link` instead. See `wp_localize_jquery_ui_datepicker()` in this file as an example.